src/components/menu_item_button/menu_item_button.ts

```diff
@@ -0,0 +1,99 @@
+import { Component } from "@odoo/owl";
+import { createMenuItem, MenuItemSpec } from "../../registries/menu_items_registry";
+import { SpreadsheetChildEnv } from "../../types";
+import { css, cssPropertiesToCss } from "../helpers";
+
+const FONT_COLOR = "#4a4f59";
+const ACTIVE_BG_COLOR = "#E6F4EA";
+const ACTIVE_FG_COLOR = "#188038";
+const HOVERED_BG_COLOR = "#f6f7fa";
+const HOVERED_FG_COLOR = "#000000";
+
+//------------------------------------------------------------------------------
+
+css/* scss */ `
+  .o-tool {
```

Comment:
We probably should have a better class name than `tool`, it doesn't make that much sense in a `MenuButton` component